### DISCUSSION TITLE
Do we have any Grafana Dashboard example on top of TEI metrics?

### DISCUSSION THREAD

**Discussion by fhkingma** [huggingface/text-embeddings-inference#158] [Ideas]
@OlivierDehaene, [similar as you shared for TGI](https://github.com/huggingface/text-generation-inference/discussions/1127#discussioncomment-7854661), does Hugging Face already implement a Grafana dashboard that you are willing to share?

**Comment by francescov1** [comment]
@fhkingma As a workaround, I just took the TGI dashboard and swapped whatever equivalents there are for the TEI. Not perfect but does give a decent bit of observability. Would love a more official dashboard though
